pkg/core/block/header.go

```diff
@@ -149,6 +156,7 @@ func (b Header) MarshalJSON() ([]byte, error) {
 		PrevHash:      b.PrevHash,
 		MerkleRoot:    b.MerkleRoot,
 		Timestamp:     b.Timestamp,
+		Nonce:         strconv.FormatUint(b.Nonce, 16),
```

Comment:
I'm not what string will C# produce for `b.Nonce == 0xff`, will it be "ff" or "00000000000000ff".

Reply:
It is `00000000000000ff` on C#

Reply:
Fixed.